src/View/Helper/DocBlockHelper.php

```diff
@@ -215,7 +215,7 @@ public function columnTypeToHintType(string $type): ?string
                 return '\Cake\I18n\Time';
         }
 
-        return null;
+        return 'string';
```

Comment:
```suggestion
        // Any unique or custom types will have a `string` type hint
        return 'string';
```
The docblock of this function needs to be updated to not say this is "ignored" but defaulted.